server/core/src/main/java/org/infinispan/server/core/transport/NettyTransportConnectionStats.java

```diff
@@ -31,7 +27,7 @@
 
    public NettyTransportConnectionStats(EmbeddedCacheManager cacheManager, ChannelGroup acceptedChannels, String threadNamePrefix) {
       this.cacheManager = cacheManager;
-      this.isGlobalStatsEnabled = cacheManager != null && SecurityActions.getCacheManagerConfiguration(cacheManager).jmx().enabled();
+      this.isGlobalStatsEnabled = cacheManager != null;
```

Comment:
I don't see how this could be false, at the very least it shouldn't be named `isGlobalStatsEnabled` unless it uses `GlobalConfiguration.statistics().enabled()`

Reply:
Oops, looks like the cache manager can be `null`, at least the way we start the server in our JCache remote TCK module:

```
2020-08-28 04:50:13,970 FATAL (main) [org.infinispan.SERVER] ISPN080028: Infinispan Server failed to start java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:395)
	at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:1999)
	at org.infinispan.server.Bootstrap.runInternal(Bootstrap.java:140)
	at org.infinispan.server.tool.Main.run(Main.java:98)
	at org.infinispan.server.Bootstrap.main(Bootstrap.java:40)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.infinispan.server.loader.Loader.run(Loader.java:76)
	at org.infinispan.server.loader.Loader.main(Loader.java:39)
Caused by: java.lang.NullPointerException
	at org.infinispan.server.core.transport.NettyTransportConnectionStats.<init>(NettyTransportConnectionStats.java:32)
	at org.infinispan.server.core.transport.NettyTransport.<init>(NettyTransport.java:76)
	at org.infinispan.server.router.router.impl.singleport.SinglePortEndpointRouter.start(SinglePortEndpointRouter.java:44)
	at org.infinispan.server.Server.run(Server.java:383)
	... 9 more
```